adapters/sqltimeout/sqltimeout_test.go

```diff
@@ -1,11 +1,12 @@
 package sqltimeout_test
 
 import (
-	"github.com/luno/workflow/adapters/sqltimeout"
 	"testing"
 
 	"github.com/luno/workflow"
 	"github.com/luno/workflow/adapters/adaptertest"
+
```

Comment:
```suggestion
```

Reply:
These are separate modules 😄

Reply:
yes but `github.com/luno/workflow/adapters/adaptertest` is part of the `github.com/luno/workflow` mod and the line is automatically created on precommit hooks

Reply:
Sorry not sure I follow, so are you saying if you have:
```go
"github.com/luno/workflow"
"github.com/luno/workflow/adapters/adaptertest"
"github.com/luno/workflow/adapters/sqltimeout"
```
Then precommit changes it to:
```go
"github.com/luno/workflow"
"github.com/luno/workflow/adapters/adaptertest"

"github.com/luno/workflow/adapters/sqltimeout"
```
?

my suggestion was just to remove the empty line in between `"github.com/luno/workflow/adapters/adaptertest"` and `"github.com/luno/workflow/adapters/sqltimeout"`